styles/globals.css

```diff
@@ -21,3 +21,8 @@ th{
 {
     background: none;
 }
+@media print {
```

Comment:
I have no idea why you added this, makes no sense. Remove it.

Reply:
I have added this because when we print the page we will get the print button and scroll to top button in pdf. So, In order to stop this from happening we need to use @media print

Reply:
printi and scroller classes are the names of buttons

Reply:
And we want to hidden this buttons only when we are printing.
That's the reason of using @media print

Reply:
Resolve the same to the print button because now if you click on print, it's visible, and add a comment in the global.css file mentioning what the `@media print` does.

Reply:
When we click on print in the pdf preview button in not visible in the latest deployment

Reply:
@hemanth-kotagiri Can you check now?